Microsoft.Toolkit.Forms.UI.XamlHost/DpiScalingPanel.cs

```diff
@@ -4,13 +4,15 @@
 
 using System;
 using System.Diagnostics;
+using foundation = Windows.Foundation;
+using uwpXaml = Windows.UI.Xaml;
```

Comment:
NIT: This naming feels strange to me

Reply:
That's because most files do have "using System;" and this makes a conflict happen between global::System.Windows.* and global::Windows.*
Is there another way to solve this conflicts? I don't think there is. If the problem is just the naming, we could change it to something better.